draft-ietf-quic-recovery.md

```diff
@@ -110,6 +110,13 @@ important to the loss detection and congestion control machinery below.
 * Retransmittable packets are those that contain at least one retransmittable
   frame.
 
+* PADDING frames do cause packets containing them to count towards bytes in
+  flight, but do not instigate acknowledgement.
```

Comment:
Instigate feels like an unusual word choice here.  Not necessarily incorrect, but unusual.

Reply:
What Mike said, but I think that it would be better to concentrate on the more important distinctions and perhaps rewrite this bit.  Your bullet list is now 6 items, which is too long anyway, so perhaps this could be reduced:

QUIC packets can contain multiple frames of different types.  The recovery mechanisms ensure that frames that need reliable delivery are acknowledged and sent in new packets as necessary.  The types of frames contained in a packet can affect recovery and congestion control logic:

* All packets are acknowledged, though packets that contain only ACK, ACK_ECN, and PADDING frames are not acknowledged immediately.

* Packets that contain CRYPTO frames are critical to the performance of the QUIC handshake and use shorter timers for acknowledgement and retransmission.

* Packets that contain only ACK and ACK_ECN frames do not count toward congestion control limits (see {{blah}}).  That is, they do not contribute toward bytes in flight.  Note that this means that PADDING frames cause packets to contribute toward bytes in flight without directly causing an acknowledgment to be sent. 

(This last point might be accomplished better with a forward reference, but I didn't see a nice place to put that, the definition of bytes in flight is buried a little.)

Reply:
Bump for this comment.  I think that it might be a better framing still, after some adjustments.